amundsen_application/static/js/ducks/tableMetadata/reducer.ts

```diff
@@ -227,6 +243,11 @@ export function getPreviewDataSuccess(
 
 /* REDUCER */
 export interface TableMetadataReducerState {
+  dashboards?: {
```

Comment:
Related dashboard information is optionally stored as its own object on the table metadata reducer state.